WebRoot/templates/prefs/Pages.template

```diff
@@ -1620,6 +1620,10 @@ NOTE: The tables inside of the filter rule sections MUST contain a tbody element
 							<td class='ZOptionsLabel'><$=ZmMsg.defaultApptDuration$></td>
 							<td class='ZOptionsField'><table role="presentation"><tr><td><select id='${id}_CAL_DEFAULT_APPT_DURATION' tabindex=0></select></td><td style="padding-left:4px;"><$=ZmMsg.minutes$></td></tr></table></td>
 						</tr>
+						<tr>
+							<td class='ZOptionsLabel'><$=ZmMsg.defaultApptDuration$></td>
```

Comment:
Is `ZmMsg.defaultCalendarScaling` correct?

Reply:
Fixed.